draft-steele-cose-hash-envelope.md

```diff
@@ -32,38 +32,41 @@ informative:
 
 --- abstract
 
-This document defines new COSE header parameters in support of a mechanism that uses hashes of detached payload as the COSE payload, which enables faster signature validation for detached payload as the availability of the original payload is not required for signature validation.
-Additionally, hints about the detached payload's content format and availability are defined.
+This document defines new COSE header parameters supporting of hashes within the payload, representing a detached COSE payloads.
```

Comment:
```suggestion
This document defines new COSE header parameters for signaling that a payload is the output of a hash function.
```

Reply:
It likely needs a bit more tweaks for readability, but it's more accurate so merged into the PR for content tweaking later.